modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/carousel.scss

```diff
@@ -0,0 +1,86 @@
+.carousel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    max-width: 800px;
+    margin: auto;
+  }
+  
+  .main-image-container {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 10px;
+  }
+  
+  .main-image {
+    width: 100%;
+    max-height: 500px;
+    object-fit: cover;
+  }
+  
+  .thumbnail-wrapper {
+    width: 100%;
+    background-color: black;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+  }
+  
+  .nav-buttons {
```

Comment:
All the classes in this CSS file are in root and have generic names, so they are very likely to spill over to the rest of the page. Please put the implementation, along with all the styles, into a very specific container, named something like `fds-carousel-view-sample`.